test/sparse/spqr.jl

```diff
@@ -27,34 +60,18 @@ for eltyA in (Float64, Complex{Float64})
         @test A\B ≈ Array(A)\B
         @test_throws DimensionMismatch A\B[1:m-1,:]
         @test A[1:9,:]*(A[1:9,:]\ones(eltyB, 9)) ≈ ones(9) # Underdetermined system
-
-        if eltyA == eltyB # promotions not defined for unexported methods
-            @test qrfact(sparse(eye(eltyA, 5)))\ones(eltyA, 5) == ones(5)
-            @test_throws ArgumentError SPQR.factorize(SPQR.ORDERING_DEFAULT, SPQR.DEFAULT_TOL, CHOLMOD.Sparse(sparse(eye(eltyA, 5))))
-            @test_throws ArgumentError SPQR.Factorization(1, 1, convert(Ptr{SPQR.C_Factorization{eltyA}}, C_NULL))
-            F = qrfact(A)
-            @test size(F) == (m,n)
-            @test size(F, 1) == m
-            @test size(F, 2) == n
-            @test size(F, 3) == 1
-            @test_throws ArgumentError size(F, 0)
-
-            # low level wrappers
-            @test_throws DimensionMismatch SPQR.solve(SPQR.RX_EQUALS_B, F, CHOLMOD.Dense(B'))
-            @test_throws DimensionMismatch SPQR.solve(SPQR.RTX_EQUALS_B, F, CHOLMOD.Dense(B))
-            @test_throws DimensionMismatch SPQR.qmult(SPQR.QX, F, CHOLMOD.Dense(B'))
-            @test_throws DimensionMismatch SPQR.qmult(SPQR.XQ, F, CHOLMOD.Dense(B))
-            @test A\B ≈ SPQR.backslash(SPQR.ORDERING_DEFAULT, SPQR.DEFAULT_TOL, CHOLMOD.Sparse(A), CHOLMOD.Dense(B))
-            @test_throws DimensionMismatch SPQR.backslash(SPQR.ORDERING_DEFAULT, SPQR.DEFAULT_TOL, CHOLMOD.Sparse(A), CHOLMOD.Dense(B[1:m-1,:]))
-        end
     end
 end
 
-# Issue 14134
```

Comment:
Yes. The test is related to the handling of pointers to SPQR objects but with this PR, we don't have such pointers anymore. All the data is now stored in Julia structs.